go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
 	github.com/go-openapi/spec v0.19.2
-	github.com/noobaa/noobaa-operator/v2 v2.0.8
+	github.com/noobaa/noobaa-operator/v2 v2.0.6-0.20200219114615-8ae457e831e4
```

Comment:
Is this noobaa-operator 2.1 ?

Reply:
2.1 is not released yet. I point to the 2.1 branch for now and will update once it will be released